lib/cookieConsent/index.js

```diff
@@ -172,12 +184,25 @@ var cookieConsent = (function () {
     };
   }
 
+  var subscribeConsent = function (callback) {
+    if (typeof callback !== 'function') {
+      throw new TypeError('cookieConsent (subscribe): callback should be a function.');
+    }
+    storeCallback = callback;
+    return {
+      unsubscribe: function () {
+        storeCallback = undefined
+      }
+    }
+  }
```

Comment:
Rename to `subscribeToConsentStore`
It should handle multiple subscribers
Write test cases to check the new function

Reply:
Created the subscribeToConsent Store